src/@chakra-ui/gatsby-plugin/components/Tag/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  createMultiStyleConfigHelpers,
+  theme,
+  defineStyle,
+} from "@chakra-ui/react"
+import { tagAnatomy } from "@chakra-ui/anatomy"
+import { $badgeColor, STATUS_COLORS } from "./utils"
+import { defineMergeStyles } from "../components.utils"
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(tagAnatomy.keys)
+
+const { Tag: tagTheme } = theme.components
+
+const baseStyleContainer = defineMergeStyles(tagTheme.baseStyle?.container, {
+  border: "1px",
+  borderColor: "transparent",
+  gap: 1,
+  borderRadius: "full",
+  px: 2,
+  minH: 8,
+  fontWeight: 300,
+  "&:any-link": {
+    textDecor: "none",
+    _focusWithin: {
+      outline: "4px solid",
+      outlineColor: "transparent",
+      outlineOffset: 0,
+    },
+  },
+})
+
+const baseStyleLabel = defineStyle({
+  ...tagTheme.baseStyle?.label,
+  fontSize: "xs",
+  textTransform: "uppercase",
+  textAlign: "center",
+})
+
+const baseStyleCloseButton = defineStyle({
+  ...tagTheme.baseStyle?.closeButton,
+  opacity: 1,
+})
+
+const baseStyle = definePartsStyle({
+  container: baseStyleContainer,
+  label: baseStyleLabel,
+  closeButton: baseStyleCloseButton,
+})
+
+const variantSubtle = defineStyle((props) => {
+  const { status = "normal" } = props
+  const defaultStyles = tagTheme.variants?.subtle(props)
+  const statusStyles = STATUS_COLORS[status].subtle
+  return {
+    container: {
+      ...defaultStyles?.container,
+      // Remove default dark mode styles
+      _dark: {},
+      ...statusStyles,
+    },
+    label: {
+      // ...warningLabelStyles
+    },
```

Comment:
I think we also need to style the `closeButton`s pseudo classes with the same styles that the container has. I'm talking specifically about the focus state.

Right now we are displaying the default Chakra styles here. I think we should use the same border styles and colors that we use for the container.

This is how it looks now when you focus on the close button:
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/c6313a91-a64d-4d95-94bb-dfcd4b6e4e37)

Not saying that this is the code but could be something like:
```tsx
closeButton: {
  _focusVisible: {
    ...statusStyles["&:any-link"]._focusWithin,
    bg: "transparent",
    boxShadow: "none",
  },
},
```

Reply:
@pettinarip This example you gave will actually work here.

I think what I am missing here is whether or not this button will actually be visible if the tag itself is a link. That might change how I approach this to ensure the focus ring for the button has enough contrast to the background of the tag (looking at the `normal` tag)

@nloureiro I don't believe we received that clarity in the DS yet. 😄 

Will the close button ever be used if the tag is a link?

Reply:
@nloureiro  Oh! I understand.

I'm going to roll with what you suggested above to provide you a proper review.

I think it will be best to have three stories here:

1. Show all tag style variants as links with text and no icons to simply demonstrate the state changes (maybe they should include the + icon as the right-side element?).
2. Provide another set of the same tags, but not as a links and containing the close button to demonstrate the buttons styling in the context of the different style variants
3. The different variants of elements that are expected to be rendered in the component.

Would this be better?

Reply:
hum... I think it might work, as far as I understand.

correct me if I'm wrong:
we will have a folder in storybook with "tags" > inside there will be 3 stories

is that it?

Reply:
Correct @nloureiro here you will see another entry
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/34d07494-e72f-4979-aa8f-5d56a8aeb068)

@TylerAPfledderer yes, make sense 👍🏼

Reply:
Sweet! I'll get that added asap

Reply:
look good to me, great job!!